Description: Get rid of the "4" in kde to make it work coherently
Origin: vendor
Author: Didier Raboud <didier@raboud.com>
Bug: http://bugs.debian.org/527340
Last-Update: 2009-05-18

--- a/kcm_gtk/kcmgtk.cpp
+++ b/kcm_gtk/kcmgtk.cpp
@@ -37,7 +37,7 @@
 #include <sys/stat.h>
 
 
-const QString KcmGtk::k_gtkRcFileName(QDir::homePath() + "/.gtkrc-2.0-kde4");
+const QString KcmGtk::k_gtkRcFileName(QDir::homePath() + "/.gtkrc-2.0-kde");
 const QString KcmGtk::k_envFileName(KGlobal::dirs()->localkdedir() + "/env/gtk-qt-engine.rc.sh");
 const QString KcmGtk::k_qtThemeName("Qt4");
 
@@ -112,7 +112,7 @@ void KcmGtk::updateFontPreview()
 
 void KcmGtk::save()
 {
-	// Write ~/.gtkrc-2.0-kde4
+	// Write ~/.gtkrc-2.0-kde
 	m_gtkRc->save();
 	
 	// Write ~/.kde/env/gtk-qt-engine.rc.sh
